### ISSUE TITLE
Onboarding tutorials/notebooks

### ISSUE THREAD

**Issue by nilsvu** [sxs-collaboration/spectre#3898]
Shall we add some onboarding tutorials/notebooks for new students? I'm thinking some introductions to DG / spectral methods, numerical derivatives, time steppers, etc to introduce/motivate our numerical methods, e.g., evolve a scalar wave in a few lines of Python and then work our way up to evolving it in SpECTRE. So the idea would be to tell a new (summer/undergrad/graduate-)student, who may have had little contact with computational physics thus far, to work through these couple of notebooks in their first week to get started. Thoughts?

**Comment by knelli2** [comment]
Yes I think this would be a very good idea, coming from somebody who was (and still kind of is) unfamiliar with the numerical methods you listed above. I like the transition from something simple that most students can understand (like python code) to how we do things in SpECTRE in a certain number of steps. When first starting out, I found the [explanation](https://spectre-code.org/databox_foundations.html) of our DataBox to be very illustrative of how you can use a simple concept and transform it into how we do things in SpECTRE.

If we do this, I think this should be a completely different section from the dev guide and the user tutorials. Maybe something like `Getting started with numerical methods in SpECTRE` or `Introduction to numerical methods in SpECTRE`
